src/profile-logic/import/linux-perf.js

```diff
@@ -10,7 +10,13 @@
  */
 export function isPerfScriptFormat(profile: string): boolean {
   const firstLine = profile.substring(0, profile.indexOf('\n'));
-  return /^(\S.+?)\s+(\d+)\/*(\d+)*\s+([\d.]+)/.test(firstLine);
+  //          +- process name (anything before the rest of the regexp, can contain spaces)
+  //          |        +- PID/ (optional))
+  //          |        |      +- TID
+  //          |        |      |          +- unknown number between brackets (optional)
+  //          |        |      |          |          +- timestamp
+  //       vvvvv   vvvvvvvvv vvv   vvvvvvvvvvvvvv vvvvvv
+  return /^\S.+?\s+(?:\d+\/)?\d+\s+(?:\[\d+\]\s+)?[\d.]+:/.test(firstLine);
```

Comment:
I completely rewrote this regexp to make it look closer to the other regexps in the same file.